Documentation/vmware-backdoor.md

```diff
@@ -1,7 +1,35 @@
-# VMware Backdoor #
+# VMware Backdoor
+
+## Setting GuestInfo Options
+
+Guestinfo settings are informations stored in the VMX of virtual machine or in the VMX representation in host memory. The criteria for being able to read it from guest OS is to use the prefix: `guestinfo.`.
+
+These settings can be set in five main ways:
+
+* You can configure them in the OVF you want to deploy. Some middlewares will help you to customize it during the deploy of the virtual machines based on this OVF (like [vcloud director](http://blogs.vmware.com/vsphere/2012/06/leveraging-vapp-vm-custom-properties-in-vcloud-director.html)). For more information, you can have a look on the following post: [Self-Configuration and the OVF Environment](http://blogs.vmware.com/vapp/2009/07/selfconfiguration-and-the-ovf-environment.html)
+
+* You can set guestinfo keys and values from the CoreOS guest itself by using preinstalled vmwaretools command like:
+
+```sh
+/usr/share/oem/bin/vmtoolsd --cmd "info-set guestinfo.<variable> <value>"
+```
+
+* You can also set guestinfo keys and values from an ESXi Service Console:
```

Comment:
This isn't specific to the ESXi Service Console. It's just a standard VIX operation. Workstation can also use these commands.